src/Behat/Testwork/Exception/FatalThrowableError.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Behat Testwork.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Testwork\Exception;
+
+use ErrorException;
+use ParseError;
+use Throwable;
+use TypeError;
+
+class FatalThrowableError extends ErrorException
```

Comment:
Please keep the ``@author`` annotation with the original author (i.e. @nicolas-grekas IIRC)